src/shared/constants/features.ts

```diff
@@ -1,6 +1,6 @@
 import {CheckOutlined, CloudUploadOutlined, CodeTwoTone, UnorderedListOutlined} from '@ant-design/icons';
 
-import {Feature} from '../models';
+import {Feature} from '@shared/models/features';
```

Comment:
In case if we change the structure some day, this feels safer. Safety was the only reason I did that for, to make it dynamic in case if we move files somewhere else (what if we change the folders' structure). @topliceanurazvan